src/components/ModerationInfoModal.tsx

```diff
@@ -42,9 +50,14 @@ export default function ModerationInfoModal(props: IPopUp) {
             .
           </p>
         </div>
-        <div className="mt-5 text-center">
+        <div className="mt-3 text-center">
           <Link to="/terms-of-service">Read our terms of service</Link>
         </div>
+        {!props.closeButton && (
+          <div className="text-center">
+            <Button onClick={acknowledgePolicy}>I understand</Button>
+          </div>
+        )}
```

Comment:
It'd be nice if there were a little room around the TOS link:
<img width="293" alt="image" src="https://github.com/ProjectProtocol/project-protocol-web/assets/6488787/c5091966-2eb7-49ff-8b32-d36183cb5774">

Since there is already a container div wrapping the other content with `vertical-rhythm`, you might just try moving both of these divs up to be children of that div so everything gets the same vertical spacing, and remove the `mt-3` class here. Alternatively, you could change `mt-3` to `my-3`, which adds top and bottom spacing, but I'd prefer the former if it looks ok to you, so that everything is uniform.

Reply:
<img width="429" alt="截屏2024-07-09 上午10 28 22" src="https://github.com/ProjectProtocol/project-protocol-web/assets/5348488/95bab0c4-6130-46eb-89f6-46bfec6e9d37">